example.rules

```diff
@@ -0,0 +1,9 @@
+# The count of CPUs per node, useful for getting CPU time as a percent of total.
+instance:node_cpu:count = count(node_cpu{mode="idle"}) without (cpu,mode)
+
+# CPU in use by CPU.
+instance_cpu:node_cpu_not_idle:rate2m = sum rate(node_cpu{mode!="idle"}[2m])) without (mode)
+
+# CPU in use by mode.
+instance_mode:node_cpu_not_idle:rate2m = sum rate(node_cpu{mode!="idle"}[2m])) without (cpu)
```

Comment:
That's not what this label filter is for.  This is specific to the functionality of node_cpu.